compiler/circlechef/core/src/ModelChef.cpp

```diff
@@ -185,11 +188,14 @@ gather_builtincode_set(const ::circlechef::ModelRecipe &model_recipe)
       auto op_chef = op_chef_registry().lookup(operation.type()).create(&operation);
       if (op_chef->code() == circle::BuiltinOperator_CUSTOM)
         continue;
-      builtin_set.insert(op_chef->code());
+
+      if (builtin_map.find(op_chef->code()) == builtin_map.end() ||
+          builtin_map[op_chef->code()] < operation.version())
+        builtin_map[op_chef->code()] = operation.version();
```

Comment:
This will permit only maximum number of version per OpCode if  I read your code correctly.
Is there no model that has same OpCode with different version?

Reply:
As I said at offline, it seems to be.
I wrote some memo related with it in https://github.com/Samsung/ONE/issues/3174#issuecomment-660802695 :)